catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -873,4 +882,21 @@ public String getSortKey(String field) {
   public Stream<String> anyTextFields() {
     return anyTextFieldsCache.stream();
   }
+
+  /**
+   * Gets the metadata size limit from custom.system.properties. Defaults to 5mb limit.
+   *
+   * @return Integer metadata size limit in bytes
+   */
+  private Integer getMetadataSizeLimit() {
+    String sizeLimit = System.getProperty("metadata.size.limit");
```

Comment:
✏️ You can provide a default value with `getProperty`.

https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#getProperty-java.lang.String-java.lang.String-

Reply:
Nice catch. Makes it cleaner. Thanks